src/Workspaces/Core/Portable/FindSymbols/ReferenceLocationExtensions.cs

```diff
@@ -39,6 +39,7 @@ internal static class ReferenceLocationExtensions
                         AddSymbols(semanticModel, documentGroup, result);
                     }
 
+                    // Keep compilation alive so that GetSemanticModelAsync remains cheap
```

Comment:
Was this intended to be included by this PR?

Reply:
Yes. Just a comment update based on an investigation I did while looking at some of the test runs.